composer.json

```
@@ -20,7 +20,7 @@
         "guzzlehttp/guzzle": "^6.3.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
+        "sminnee/phpunit": "^5.7",
         "squizlabs/php_codesniffer": "^3.0",
         "sminnee/phpunit-mock-objects": "^3.4.5"
```

This bit is no longer needed because `sminnee/phpunit` requires in directly.

```suggestion
        "squizlabs/php_codesniffer": "^3.0"
```